src/main/java/cz/cvut/kbss/study/persistence/dao/UserDao.java

```diff
@@ -83,9 +83,11 @@ public List<User> findByInstitution(Institution institution) {
 
     public int getNumberOfInvestigators() {
         return ((BigInteger) em.createNativeQuery(
-                                       "SELECT (count(?p) as ?investigatorCount) WHERE { ?p a ?typeDoctor . MINUS {?p a ?typeAdmin}}")
-                               .setParameter("typeDoctor", URI.create(Vocabulary.s_c_doctor))
-                               .setParameter("typeAdmin", URI.create(Vocabulary.s_c_administrator)).getSingleResult()
+                                       "SELECT (count(?p) as ?investigatorCount) WHERE { ?p a ?typeUser . MINUS {?p ?hasRoleGroup ?roleGroup . ?roleGroup ?hasRole ?typeAdmin}}")
```

Comment:
For now, this is ok, but we need to create a ticket to fix it later. We need to include only those who can fill in the records.

Reply:
Will be resolved in https://github.com/kbss-cvut/record-manager-ui/issues/202#issuecomment-2355886444